src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/LincheckFailure.kt

```diff
@@ -12,63 +12,71 @@ package org.jetbrains.kotlinx.lincheck.strategy
 import org.jetbrains.kotlinx.lincheck.*
 import org.jetbrains.kotlinx.lincheck.execution.*
 import org.jetbrains.kotlinx.lincheck.runner.*
+import org.jetbrains.kotlinx.lincheck.strategy.managed.ManagedCTestConfiguration
 import org.jetbrains.kotlinx.lincheck.trace.Trace
 
 sealed class LincheckFailure(
     val scenario: ExecutionScenario,
     val results: ExecutionResult,
-    val trace: Trace?
+    val trace: Trace?,
+    val testConfig: ManagedCTestConfiguration?
```

Comment:
Once you implement `AnalysisProfile` class, let's pass it instead of `ManagedCTestConfiguration`. 

Passing `ManagedCTestConfiguration` has the following drawbacks:
- It contains too much details not relevant to the trace representation.
- It is coupled with `ManagedStrategy`.
- We will likely change the configuration API in the future.

Reply:
Sounds good, done.